applications/fake-services/replicated/kots-app.yaml

```diff
@@ -6,7 +6,8 @@ spec:
   title: Fake Services
   icon: https://www.shareicon.net/data/256x256/2016/05/16/766096_zoo_512x512.png
   statusInformers:
-    - deployment/frontend
+    - '{{repl if ConfigOptionEquals "frontend_informer_enabled" "yessss"}}deployment/frontend{{repl end}}'
```

Comment:
If we are going to use bool type, then we need to modify this
'{{repl if ConfigOptionEquals "frontend_informer_enabled" "1"}}deployment/frontend{{repl end}}'